modules/ztp-preparing-the-ztp-git-repository-ver-ind.adoc

```diff
@@ -17,10 +17,12 @@ This means that the version of {product-title} running on the hub cluster can be
 For each version, create the following resources:
 * `kustomization.yaml` file that explicitly includes the files in that directory
 * `source-crs` directory to contain reference CR configuration files from the `ztp-site-generate` container
++
+If you want to work with user-provided CRs, you must create a separate directory for those.
```

Comment:
This is the exact same line added in the next paragraph at the last line, seems duplicate

Reply:
I updated the next paragraph to say "If you want to work with custom manifests, you must create a separate directory for those."

This step is about custom policygen CRs and the next step is about custom manifests.